### PR TITLE
add cross_entropy_loss_cn doc to functional_cn and loss_cn

### PR DESCRIPTION
**Pull request by chajchaj**
add cross_entropy_loss_cn doc to functional_cn and loss_cn

### CODE REVIEW

File: doc/fluid/api_cn/nn_cn/functional_cn/cross_entropy_loss_cn.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.. _cn_api_nn_functional_nll_loss:
```
Comment (huangjun12) [inline doc/fluid/api_cn/nn_cn/functional_cn/cross_entropy_loss_cn.rst]: change to cross_entropy_loss

File: doc/fluid/api_cn/nn_cn/loss_cn/CrossEntropyLoss_cn.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 CrossEntropyLoss
 -------------------------------
 
-.. py:function:: paddle.nn.loss.CrossEntropyLoss(weight=None, reduction='mean', ignore_index=-100)
+.. py:function:: paddle.nn.loss.CrossEntropyLoss(weight=None, ignore_index=-100, reduction='mean')
 
 该OP计算输入input和标签label间的交叉熵损失 ，它结合了`LogSoftmax` 和 `NLLLoss` 的OP计算，可用于训练一个 `n` 类分类器。
```
Comment (huangjun12) [inline doc/fluid/api_cn/nn_cn/loss_cn/CrossEntropyLoss_cn.rst]: add space before `LogSoftmax

File: doc/fluid/api_cn/nn_cn/loss_cn/CrossEntropyLoss_cn.rst [hunk 3]
```diff
@@ -1,7 +1,7 @@
 CrossEntropyLoss
```
Comment (huangjun12) [inline doc/fluid/api_cn/nn_cn/loss_cn/CrossEntropyLoss_cn.rst:3]: .. _cn_api_nn_CrossEntropyLoss:

**Comment by chajchaj** [comment]
ce1
ce2
ce3
ce4

**Review by yaoxuefeng6 — APPROVED** [review]
LGTM

File: doc/fluid/api_cn/nn_cn/functional_cn/cross_entropy_loss_cn.rst [hunk 4]
```diff
@@ -0,0 +1,51 @@
+.. _cn_api_nn_functional_cross_entropy_loss:
+
+cross_entropy_loss
+-------------------------------
+.. py:function:: paddle.nn.functional.cross_entropy_loss(input, label, weight=None, ignore_index=-100, reduction='mean')
```
Comment (jzhang533) [inline doc/fluid/api_cn/nn_cn/functional_cn/cross_entropy_loss_cn.rst:5]: 没有name 参数？

**Review by huangjun12 — APPROVED** [review]

File: doc/fluid/api_cn/nn_cn/loss_cn/CrossEntropyLoss_cn.rst [hunk 5]
```diff
@@ -1,9 +1,11 @@
+.. _cn_api_nn_loss_CrossEntropyLoss:
+
 CrossEntropyLoss
 -------------------------------
 
-.. py:function:: paddle.nn.loss.CrossEntropyLoss(weight=None, reduction='mean', ignore_index=-100)
+.. py:function:: paddle.nn.loss.CrossEntropyLoss(weight=None, ignore_index=-100, reduction='mean')
```
Comment (jzhang533) [inline doc/fluid/api_cn/nn_cn/loss_cn/CrossEntropyLoss_cn.rst:6]: name参数？

**Review by jzhang533 — APPROVED** [review]
will have another pr

**Pull request merged**